src/components/layouts/FilterBar.tsx

```diff
@@ -146,25 +146,25 @@ export const FilterBar = ({
                     placeholder="Assignee"
                     value={assigneeValue}
                     selectorType={SelectorType.ASSIGNEE_SELECTOR}
-                    //****Disabling re-assignment completely for now***
-                    // extraOption={NoAssigneeExtraOptions}
-                    // extraOptionRenderer={(setAnchorEl, anchorEl, props) => {
-                    //   return (
-                    //     noAssigneOptionFlag && (
-                    //       <>
-                    //         <ExtraOptionRendererAssignee
-                    //           props={props}
-                    //           onClick={(e) => {
-                    //             updateAssigneeValue({ id: '', name: 'No assignee' })
-                    //             setAnchorEl(anchorEl ? null : e.currentTarget)
-                    //             handleFilterOptionsChange(FilterOptions.ASSIGNEE, 'No assignee')
-                    //           }}
-                    //         />
-                    //         {loading && <MiniLoader />}
-                    //       </>
-                    //     )
-                    //   )
-                    // }}
+                    extraOption={NoAssigneeExtraOptions}
```

Comment:
tech debt: we should be sharing this code for assignee drop-down right? Lets try to refactor this when we add back re-assignment, wdyt?